lib/driver.js

```diff
@@ -218,6 +219,7 @@ class XCUITestDriver extends BaseDriver {
         shouldUseCompactResponses: DEFAULT_SETTINGS.shouldUseCompactResponses,
         mjpegServerScreenshotQuality: DEFAULT_SETTINGS.mjpegServerScreenshotQuality,
         mjpegServerFramerate: DEFAULT_SETTINGS.mjpegServerFramerate,
+        screenshotQuality: DEFAULT_SETTINGS.screenshotQuality,
```

Comment:
I think we should not repeat the same defaults in WDA and inside the driver. Lets only pass the setting, which value was atcually changed.

Reply:
that was reasonable before we had our own wda fork

Reply:
@imurchie @jlipps What do you think?